src/dsp_tools/commands/start_stack.py

```diff
@@ -180,7 +180,7 @@ def _wait_for_fuseki(self) -> None:
                 )
                 if response.ok:
                     break
-            except Exception:  # pylint: disable=broad-exception-caught
+            except Exception:  # noqa: BLE001 (blind-except)
```

Comment:
should those indeed be ignored? I'm not convinced

Reply:
What would your suggestion then be how to refactor the code? I'm afraid of catching a more specific exception class here...

Reply:
Looking at the code now: why do we even retry here, if we call `http_call_with_retry` anyways?

Reply:
`http_call_with_retry` catches only timeout errors. I'm not sure, but perhaps we get an error as long as fuseki is not up yet. As soon as it's up, we `break` from the loop.

Reply:
just seems counter-intuitive to have a retry mechanism around a retry mechanism... but let's not get caught up on this one. It is a detail after all!